lib/services/route.dart

```diff
@@ -22,17 +20,18 @@ class HikingRoute {
     this.pointsOfInterest = pointsOfInterest;
     this.elevations = elevations;
     this.date = DateFormat('yyyy-MM-dd HH:mm').format(DateTime.now());
-    this.title = 'Sample title'; // TODO get region or custom info as title
+    setTitle('');
   }
 
   Future<String> buildTitle() async{
    List<Address> results = await Geocoder.local.findAddressesFromCoordinates(Coordinates(path[0].latitude, path[0].longitude));
```

Comment:
imo we should just use the complete addressline and cap the text when its to long with the flutter text widget overflow parameter

Reply:
In our last meeting we decided to use only street and city because there is to much information in the complete addressline

Reply:
handled by Text widget within table